tools/docker/bootstrap-mercurial.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-MERCURIAL_VERSION="6.4.5"
+MERCURIAL_VERSION="6.8.1"
 VERSION_CONTROL_TOOLS_REV="7436689a409528730ee6a7a2a5e763c9fb821a8f"
```

Comment:
We should also update this one with the latest revision from https://hg.mozilla.org/hgcustom/version-control-tools.